AirBench/Scripts/Home.js

```diff
@@ -60,6 +72,36 @@
     });
     map.on('click', clickMarker);
     //map.addEventListener('click', clickMarker);
+    function updateList(){
+        let min = document.getElementById('min').value;
+        let max = document.getElementById('max').value;
+        console.log(min, max);
+        //console.log(min, max);
+        //call api
+        //let response = await fetch("/api/bench/filtered/?min=" + min + "&max=" + max);
+        //let jsonResult = await response.json();
+        //console.log(jsonResult);
+
+        //filter table rows based on data -> id
+        let table = document.getElementById("list");
+        let rows = table.getElementsByTagName("tr");
+        for (let i=1; i < rows.length; i++){
+            let numseats =  parseInt(rows[i].getAttribute("data-numseats"), 10);
+            
+            if (min == "" && max==""){
+                rows[i].style.display = "";
+            }
+            else if (numseats >= min && numseats <= max){
+                rows[i].style.display = "";
+            } 
```

Comment:
Will you refactor this at some point to allow the user to provide either a min or max # of seats to filter by?